src/hooks/resources/device.ts

```diff
@@ -353,6 +353,19 @@ sbvrUtils.addPureHook('PATCH', 'resin', 'device', {
 				),
 			);
 		}
+		if (request.values.supervisor_version) {
```

Comment:
This would be safer as
```suggestion
		if (request.values.supervisor_version != null) {
```